core/src/main/java/io/undertow/attribute/TransportProtocolAttribute.java

```diff
@@ -24,7 +24,9 @@
  * The request method
  *
  * @author Stuart Douglas
+ * @deprecated Please use {@link RequestProtocolAttribute}
  */
+@Deprecated(forRemoval = true)
```

Comment:
We must discuss when is the best moment to drop support to jdk8. For now, we use jdk8, and the project unfortunately does not compile with this change

Reply:
Yep, forRemoval annotation attribute have been introduced since JDK9.